sunpy/net/dataretriever/tests/test_eve.py

```diff
@@ -82,6 +83,12 @@ def test_fido(query):
     assert isinstance(client, eve.EVEClient)
     response = Fido.fetch(qr)
     assert len(response) == qr._numfile
+    path1 = '.'
+    response1 = Fido.fetch(qr, path=path1)
+    assert len(response1) == qr._numfile
+    path2 = pathlib.Path('.')
+    response2 = Fido.fetch(qr, path=path2)
+    assert len(response2) == qr._numfile
```

Comment:
I wonder if Fido.fetch doesn't already have tests for this somewhere else. Not sure that this should be added just under an eve test.

Reply:
Sorry for not looking properly. Just found the real test under `net.tests.test_fido`